mfr/extensions/image/export.py

```diff
@@ -36,13 +41,18 @@ def export(self):
             else:
                 image = Image.open(self.source_file_path)
 
-            if max_size:
+            if any(size is None for _, size in max_size.items()):
```

Comment:
`max_size` is dictionary containing only two keys `w` and `h`. This `if` makes things look complicated. Why not use:

```python
if max_size.get('w') and max_size.get('h'):
    ....
```

In addition, if there is a real issue that one of the two is missing (, which I doubt), we should raise an error as well.